waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -789,7 +811,7 @@
            health-check-max-consecutive-failures health-check-port-index health-check-proto image
            mem min-instances namespace ports run-as-user]
     :as service-description}
-   {:keys [container-init-commands default-container-image default-namespace log-bucket-url image-aliases]
+   {:keys [container-init-commands default-container-image default-namespace log-bucket-url image-aliases envoy-proxy-constant]
```

Comment:
How do you feel about renaming `envoy-proxy-constant` to `reverse-proxy-flag`?

Reply:
Done